### ISSUE TITLE
Span<T> isn't available on .NET Core 2.0

### ISSUE THREAD

**Issue by AArnott** [dotnet/docs#5729]
Span<T> is new to .NET Core 2.1. This topic incorrectly claims it is available in .NET Core 2.0.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 36ef14f5-5cfc-5d83-971f-18789cf45237
* Version Independent ID: 9aa52936-b378-8715-4b31-46a1c0974c43
* Content: [Span<T> Struct (System)](https://docs.microsoft.com/en-us/dotnet/api/system.span-1?view=netcore-2.0#feedback)
* Content Source: [xml/System/Span`1.xml](https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System/Span`1.xml)
* Service: **unspecified**
* Product: **.net**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **dotnetcontent**

**Comment by mairaw** [comment]
@weshaggard was Span part of extensions found in the master branch when we grabbed the assemblies for 2.0? I remember having discussed this issue before, I just don't remember if we found a way to solve this.

**Comment by AArnott** [comment]
A related problem exists for [ReadOnlySpan<T>.DangerousGetPinnableReference()](https://docs.microsoft.com/en-us/dotnet/api/system.readonlyspan-1.dangerousgetpinnablereference?view=netcore-2.0&viewFallbackFrom=netcore-2.1) which interestingly enough is documented as *only* being available on .NET Core 2.0 (which it is not available on), and that it is not available on .NET Core 2.1 (which is where it was introduced).

**Comment by BillWagner** [comment]
@AArnott My hunch is that @mairaw is correct tat the docs were generated from extensions for .NET Core 2.0. The `ReadOnlySpan<T>.DangerousGetPinnableReference()` API changed names to [GetPinnableReference](https://docs.microsoft.com/en-us/dotnet/api/system.readonlyspan-1.getpinnablereference?view=netcore-2.1#System_ReadOnlySpan_1_GetPinnableReference)  See #5845 and #5767 for details.

**Comment by Mart-Bogdan** [comment]
@AArnott  you have to install package `System.Memory` as I've found out just now.

**Comment by AArnott** [comment]
Yes, I've learned that recently as well. Can the docs mention this? The excellent apisof.net site mentions packages that must be installed like this.

**Comment by BillWagner** [comment]
Issue #8192 has ongoing discussions on how to ensure this kind of information is readily available.

**Comment by markusschaber** [comment]
@Mart-Bogdan Additionally, Assembly names and NuGet-Packages are not always matching 1:1.

**Comment by AArnott** [comment]
Knowing the minimum version that adds the type to the package for a given platform is also important.
